README.md

```diff
@@ -4,33 +4,40 @@ Service to send delta events from CHIPS on the correct kafka topic
 Environment Variables
 -----------------
 
-|  Variable                         |  Example                          |  Description                                       |
-| --------------------------------- | --------------------------------- | -------------------------------------------------- |
-| BIND_ADDR                         | 5010                              | Bind Address                                       |
-| KAFKA_BROKER_ADDR                 | chs-kafka:9092                    | Kafka broker address (can be comma separated)      |
-| SCHEMA_REGISTRY_URL               | http://chs-kafka:8081             | Schema registry URL                                |
-| OFFICER_DELTA_TOPIC               | officers-delta                    | Kafka topic to write messages to                   |
-| OPEN_API_SPEC                     | ./apispec/apispec.yml             | OpenAPI schema location                            |
-
+|  Variable                         |  Example                          |  Description                                       |  Required        | Default value |        
+| --------------------------------- | --------------------------------- | -------------------------------------------------- | ---------------- | ------------- |
+| BIND_ADDR                         | 5010                              | Bind Address / application port                    | TRUE             |               |
+| KAFKA_BROKER_ADDR                 | chs-kafka:9092                    | Kafka broker address (can be comma separated)      | TRUE             |               |
```

Comment:
I feel it will be more clearer to wider audience if we change the Required values to `YES` or `NO` compared to `TRUE` or `FALSE`

Reply:
Added, good suggestion thank you!